src/sentry/integrations/base.py

```diff
@@ -463,3 +468,72 @@ def notify_remove_external_team(self, external_team: ExternalActor, team: Team)
 
     def remove_notification_settings(self, actor_id: int, provider: str) -> None:
         pass
+
+
+def is_response_success(resp) -> bool:
+    if resp.status_code and resp.status_code < 300:
+        return True
+    return False
+
+
+def is_response_error(resp) -> bool:
+    if not resp.status_code:
+        return False
+    return resp.status_code >= 400 and resp.status_code != 429 and resp.status_code < 500
+
```

Comment:
Do you want to retain the functionality of checking for fatal errors? It's pretty specific to avoid false positives https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/slack/client.py#L74-L82

Reply:
oh nvm I see it below

Reply:
i'm retaining it here https://github.com/getsentry/sentry/pull/72348/files#diff-06439c73605aa1447ba2ca4f2dfe7bb2b579dad459893ace24dd7252f5003355R42-R46, i moved it out of the old SlackClient